htdocs/core/class/commonobject.class.php

```diff
@@ -9797,7 +9797,7 @@ public function show_photos($modulepart, $sdir, $size = 0, $nbmax = 0, $nbbyrow
 		 $filearray=array_merge($filearray, $filearrayold);
 		 }*/
 
-		completeFileArrayWithDatabaseInfo($filearray, $relativedir);
```

Comment:
Why not adding $this as parameter (so we provide all the object). We wil be able to retreive $objectfordbinfo->entity and we will have all other data of object for future need.

Reply:
i did the modification, is it ok for you ?